waspc/data/Generator/templates/sdk/wasp/core/serialization.ts

```diff
@@ -1,3 +1,31 @@
+import { Prisma } from '@prisma/client'
+import type { Decimal as DecimalType } from "@prisma/client/runtime/library"
+import SuperJSON from 'superjson'
```

Comment:
Can we import just the stuff we need?

```typescript
import { SuperJSON, serialize, deserialize } from 'superjson'
```

We usually try to avoid default exports and improts, reasoning here: #546

Plus, if we do it like this, we won't have to deal with bindings and `this` shennanigans - Superjson already did that for us.

Reply:
This and the re-binding stuff was because I was afraid that the custom class registry was per-instance and would not apply to the default thing, but I saw now that inside their code they always use a default instace, so we can indeed be simpler.

Reply:
Wow, nice thoroughness! 

I admit that didn't occur to me at all. Glad it turned out to be possible :)